cli/testdata/spinner_cases/change_message.ts

```diff
@@ -7,4 +7,4 @@ spinner.start();
 setTimeout(() => (spinner.message = "One dino 🦕"), 125); // 150
 setTimeout(() => (spinner.message = "Two dinos 🦕🦕"), 200); // 225
 
-setTimeout(spinner.stop, 500);
+setTimeout(() => spinner.stop(), 500);
```

Comment:
Isn't this the same?

Reply:
No. The existing call doesn't have access to `this` inside the method, and errors at `this.#intervalId`

Reply:
Interesting... That's what's blocking #5497.